src/app.js

```diff
@@ -0,0 +1,29 @@
+export default {
+  name: "InputFormats",
+  components: {
+    VueMoney: () => import("./components/money"),
+  },
+  props: {
+    type: {
+      type: String,
+      default: "money",
+    },
+  },
+  data() {
+    return {};
+  },
+
+  render(h) {
+    let template;
+
+    switch (this.type) {
+      case "money":
```

Comment:
use consts. Code could be changed to something like: `ComponentType.Money`. The idea behind this is, whenever we want we can change once the name and it would be changed everywhere.